gpu/api.h

```diff
@@ -25,6 +25,7 @@ extern "C" {
 #define	SPACEMESH_API_ERROR_CANCELED			-4
 #define	SPACEMESH_API_ERROR_NO_COMPOTE_OPTIONS	-5
 #define	SPACEMESH_API_ERROR_INVALID_PARAMETER	-6
+#define	SPACEMESH_API_ERROR_INVALID_PROVIDER_ID	-7

```

Comment:
I had to manually update this header by copying from the `gpu-post` release zip file. This should be done automatically by calling `make get-gpu-setup`